pkg/ibmvpc/nacl_analysis.go

```diff
@@ -136,7 +134,7 @@ func (r *NACLRule) dumpRule() string {
 	return fmt.Sprintf("src: %s, dst: %s, conn: %s, action: %s", r.src.ToIPRanges(), r.dst.ToIPRanges(), r.connections.String(), r.action)
 }
 
-func (na *NACLAnalyzer) dumpNACLrules() string {
+func (na *NACLAnalyzer) DumpNACLrules() string {
```

Comment:
Why exporting all these dumping functions? Is it easier during debugging?

Reply:
They are currently not used but I still want to keep them, and when exporting there are no warnings for those unused functions.

Reply:
I would prefer not to export them. Before you even notice, someone might consume this library and actually rely on the output of these functions, and then you have to maintain them forever :wink:

Either add a lint exception or add something like
```golang
var _ = (*ResourcesContainer).printDetails // avoiding "unused" warning
```

Reply:
ok, changed accordingly